### ISSUE TITLE
Windows Installation Issue

### ISSUE THREAD

**Issue by andrearosasco** [open-mmlab/mmcv#1509]
Hello, how do I install mmcv-full on windows? I looked up [https://download.openmmlab.com/mmcv/dist/cu111/torch1.8.0/index.html](https://download.openmmlab.com/mmcv/dist/cu111/torch1.8.0/index.html
) and all the wheels are for manylinux.

**Comment by zhouzaida** [comment]
Yet, currently, we only provide the pre-built packages for linux. You can refer to https://mmcv.readthedocs.io/en/latest/get_started/build.html#build-on-windows to build mmcv-full from source.

**Comment by andrearosasco** [comment]
Thanks, I've managed to install it and now everything is working fines. I've had some minor issues with some libraries and executables not being found during the compilation but I was able to solve them by updating LIB, INCLUDE and PATH environment variable everytime I run into a "ddl xyz.lib not found".  

I don't know if these issues are common but if they are it may be worth to write about it in the installation guide.
